src/app/views/forgotten-password/forgottenPasswordController.jsx

```diff
@@ -75,13 +79,15 @@ export class ForgottenPasswordController extends Component {
                 if (error.body != null && error.body.errors != null) {
                     error.body.errors.forEach(anError => {
                         errorContents = this.showValidationError(anError, errorContents);
-                        validationErrors.heading = "Fix the following: ";
                     });
                 } else {
                     this.notifyUnexpectedError();
                 }
 
-                validationErrors.body = errorContents.errorsForBody;
+                if (errorContents.errorsForBody.length >= 1) {
```

Comment:
we should capture the error with email/userId for logging purpose

Reply:
The error is captured on Florence Serverside (within this repo, it will output to our centralised logging system which can then be accessed in Kibana). Once server-side error has been logged it then passes sends the clientside a response which is what we see here so that the user can see something went wrong. In the case of a bad email/password these are handled in the showValidationError switch statements.